addons/medical/scripts/Game/ACE_Medical/Character/SCR_CharacterControllerComponent.c

```diff
@@ -5,54 +5,77 @@ modded class SCR_CharacterControllerComponent : CharacterControllerComponent
 {
 	protected SCR_CharacterDamageManagerComponent m_pACE_Medical_DamageManager;
 	protected const float ACE_MEDICAL_SECOND_CHANCE_DEACTIVATION_TIMEOUT_MS = 1000;
-	
+
 	//------------------------------------------------------------------------------------------------
-	//! Initialize member variables
+	//! Initialize member variables.
```

Comment:
None of the other doc blocks end with a punctuation, keep consistent
```suggestion
	//! Initialize member variables
```